### PR TITLE
New device Cisco Air

### PR DESCRIPTION
**Pull request by mpikzink**
Add Cisco AIR-CAP2702,AP1142,2602 inclusive connected users.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/OS/Ciscoair.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Ios.php
```
Comment (laf) [inline LibreNMS/OS/Ciscoair.php]: Can you update the comments to be Ciscoair.

File: includes/definitions/ciscoair.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+os: ciscoair
+text: 'Cisco IOS'
```
Comment (laf) [inline includes/definitions/ciscoair.yaml]: Do these devices run Cisco IOS?

Comment (mpikzink) [reply includes/definitions/ciscoair.yaml]: Yes, it does.

#show version
Cisco IOS Software, C1140 Software (C1140-K9W7-M), Version 15.3(3)JD11, RELEASE SOFTWARE (fc1)

File: LibreNMS/OS/Ciscoair.php (hunk 1 above)
Comment (mpikzink) [reply LibreNMS/OS/Ciscoair.php]: Thanks for the information.

**Comment by Jellyfrog** [comment]
@mpikzink the tests fails. Can you look into this please?

**Comment by mpikzink** [comment]
@Jellyfrog Sorry for the delay. I will have a look at the fail.
@murrant Yes in pricipal it's ios but accesspoints and the ios class has no wireless discovery and poller so I wanted wo extend the ios class. Should I better add the poller to the ios class? I am not sure because wireless is not the typical thing for Cisco.

**Comment by Jellyfrog** [comment]
Closing due to inactivity, feel free to reopen!

**Comment by murrant** [comment]
@mpikzink It should be ios, check if the device is an air device, and then poll wireless  sensors.  see the Ios.php class

**Pull request closed without merging**